_example/helper/field.go

```diff
@@ -165,16 +178,25 @@ func FieldToMap(model interface{}, fields map[string]interface{}) map[string]int
 
 		if contains(fields, jsonKey) {
 			v := fields[jsonKey]
+			var err error
```

Comment:
@dtan4 The following code was warned:

```go
u[jsonKey], err := FieldToMap(vs.Field(i).Elem().Interface(), v.(map[string]interface{}))
// => Warning: Go fails to infer type in assignment: “non-name on left side of :=”
```

The left-hand-side of := must be a list of identifiers, but `u[jsonKey]` is not identifiers.
So I can't use the shorthand declaration.